homeassistant/auth/models.py

```diff
@@ -27,6 +28,10 @@
 TOKEN_TYPE_LONG_LIVED_ACCESS_TOKEN = "long_lived_access_token"
 
 
+class AuthFlowResult(BaseFlowResult[tuple[str, str]], total=False):
+    """Typed result dict for auth flow."""
```

Comment:
Does it make sense to convert this to a type alias? Probably depends on if you plan to add additional keys.
```py
AuthFlowResult = BaseFlowResult[tuple[str, str]]
```

Reply:
Yeah, that makes sense. If additional keys are needed in the future it can be made a class then 👍

Reply:
Fixed